common/src/main/java/com/tencent/rss/common/config/RssConf.java

```diff
@@ -581,6 +582,11 @@ public <T> RssConf set(ConfigOption<T> option, T value) {
     return this;
   }
 
+  public <T> RssConf setList(ConfigOption<List<T>> option, Object value) {
```

Comment:
Flink class `Configuration` don't have method `setList`. Could we imitate the Flink api?

Reply:
Let me take a look.

Reply:
You are right. I remove it and reuse the method of `RssConf.set` to set list values.